src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Formatters/IPackageReferenceContextInfoFormatter.cs

```diff
@@ -37,7 +38,9 @@ private IPackageReferenceContextInfoFormatter()
             int propertyCount = reader.ReadMapHeader();
             for (int propertyIndex = 0; propertyIndex < propertyCount; propertyIndex++)
             {
-                switch (reader.ReadString())
+                var switchKey = reader.ReadString();
```

Comment:
The name "switchKey" describes that this variable is being used in a switch but maybe we can name it something more descriptive about what it is rather than how it's used in code. What strings are we expecting to read from the reader?

Reply:
Fixed.